exonum-java-binding/core/rust/src/runtime/java_service_runtime.rs

```diff
@@ -237,3 +257,59 @@ impl JavaServiceRuntime {
         }))
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn transform_jni_error_error_type_other() {
```

Comment:
Don't you trust LLVM in making pattern matching? :)

Reply:
I do, but I don't trust myself in writing `match` blocks 😄

Reply:
I agree that it seems to be too easy to break, so we can reduce the number of tests to three (a single known error code, unknown error code, other type of error).